docs/source/DevelopmentGuide.rst

```diff
@@ -0,0 +1,313 @@
+Development Guide
+=================
+
+Ready to contribute to the project? Here is how to set up a local development environment.
+
+Initial Setup
+#############
+
+1. Fork the `cda-tum/syrec <https://github.com/cda-tum/syrec>`_ repository on GitHub (see https://docs.github.com/en/get-started/quickstart/fork-a-repo).
+
+2. Clone your fork locally
+
+    .. code-block:: console
+
+        $ git clone git@github.com:your_name_here/syrec --recursive
+
+    .. warning::
+
+        The :code:`--recursive` flag is required to also clone all the required submodules.
+        If you happen to forget passing the flag on your initial clone, you can initialize all the submodules by executing :code:`git submodule update --init --recursive` in the main project directory.
+
+3. Change into the project directory
+
+    .. code-block:: console
+
+        $ cd syrec
+
+4. Create a branch for local development
+
+    .. code-block:: console
+
+        $ git checkout -b name-of-your-bugfix-or-feature
+
+    Now you can make your changes locally.
+
+5. Building the project requires Boost library with a minimum version of *1.71.0*
```

Comment:
```suggestion
5. Building the project requires the Boost libraries with a minimum version of *1.71.0*.
```
Also add the link as in the installation instructions.

Reply:
done 👍🏻